packages/cli/src/InternalHooks.ts

```diff
@@ -202,7 +202,11 @@ export class InternalHooks {
 	}
 
 	async onWorkflowSaved(user: User, workflow: IWorkflowDb, publicApi: boolean): Promise<void> {
-		const { nodeGraph } = TelemetryHelpers.generateNodesGraph(workflow, this.nodeTypes);
+		const isCloudDeployment = config.getEnv('deployment.type') === 'cloud';
+
+		const { nodeGraph } = TelemetryHelpers.generateNodesGraph(workflow, this.nodeTypes, {
+			isCloudDeployment,
+		});
```

Comment:
Why are we filtering for cloud? The story doesn't mention this?

Reply:
this is followup to this [issue](https://linear.app/n8n/issue/AI-127/add-more-ai-node-info-to-telemetry), should be enabled on cloud only